test/urn.spec.ts

```diff
@@ -238,6 +238,15 @@ describe("Basic use cases", function () {
     type: "blockchain-collection-third-party"
   })
 
+  testValidUrnToInclude("urn:decentraland:mumbai:collections-thirdparty:aThirdParty:winterCollection", {
+    contractAddress: "0x3CFe76fbf8E2868075c10161667ab3Fea19cA8D2",
+    collectionId: "winterCollection",
+    blockchain: "ethereum",
+    thirdPartyName: "aThirdParty",
+    network: "mumbai",
+    type: "blockchain-collection-third-party"
```

Comment:
`type` should be different for third party collection and third party collection item